cpp/src/aztec3/circuits/rollup/base/native_base_rollup_circuit.cpp

```diff
@@ -116,10 +116,12 @@ NT::fr iterate_through_tree_via_sibling_path(NT::fr leaf, NT::uint32 leafIndex,
 }
 
 template <size_t N>
-void check_membership(NT::fr leaf, NT::uint32 leafIndex, std::array<NT::fr, N> siblingPath, NT::fr root)
+void check_membership(
+    DummyComposer& composer, NT::fr leaf, NT::uint32 leafIndex, std::array<NT::fr, N> siblingPath, NT::fr root)
 {
     auto calculatedRoot = iterate_through_tree_via_sibling_path(leaf, leafIndex, siblingPath);
-    // TODO: update tests to build the correct trees, and then -> assert(calculatedRoot == root);
+    // TODO: update tests to build the correct trees
+    composer.do_assert(calculatedRoot == root, "Membership check failed");
     (void)calculatedRoot;
     (void)root;
```

Comment:
We can delete these `(void)x;` lines everywhere now. They were only needed when calling `assert`, I think.